SecretParser.cs

```diff
@@ -4557,25 +4559,34 @@ public static void SetFrench()
             deAttacks.Add("Inversion de tendance");
             deAttacks.Add("Jet de pierre");
             deAttacks.Add("L'art de la guerre");
+			deAttacks.Add("Malédiction persistante");

```

Comment:
I don't believe this is required.  This section should only have skills containing the word "de"